### PR TITLE
Required minimum set of permissions for Google CPI

### PR DESCRIPTION
**Pull request by cdutra**
[#149971892](https://www.pivotaltracker.com/story/show/149971892)

### CODE REVIEW

File: google-required-permissions.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    - storage.objects.create
+    - storage.objects.delete
+    ```
+
+    and `agent-blobstore-role.yml`.
+
+    ```yaml
+    title: BOSH Agent GCS Blobstore
+    stage: BETA
+    description: Allows BOSH Agent GCS Blobstore client tp perform necessary operations to the blobstore
```
Comment (dpb587-pivotal) [inline google-required-permissions.html.md.erb]: `tp` -> `to`

File: google-required-permissions.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+4. On Google Cloud navigate go to `IAM & admin` > `Service accounts` and click on `CREATE SERVICE ACCOUNT`.
+   Give your service account a name, check `Furnish a new private key` and select the aforementioned role.
+
+   <%= image_tag("images/gcp-service-account.png") %>
+
+---
+## <a id="director-with-gcs-blobstore"></a> Google CPI and Director configured with a GCS blobstore
+
+This configuration is similar to the previous one except that it's used when the Director and the Agents use GCS as their [blobstore](bosh-components.html#blobstore) instead of an internal blobstore provided by the bosh release.
```
Comment (dpb587-pivotal) [inline google-required-permissions.html.md.erb]: Might be useful to clarify these are used in addition to the previous one. The "... is similar to the previous one ..." part makes it sound a little like this is an alternative to that.

**Comment by dpb587-pivotal** [comment]
@cppforlife it'd be nice if docs could embed this sort of file so it could be curl'd, too.

**Comment by cdutra** [comment]
Thanks @dpb587-pivotal. I'll make the changes first thing in the morning.

**Pull request merged**